hana/lib/HANAService.js

```diff
@@ -421,6 +422,11 @@ class HANAService extends SQLService {
         }
 
         if (parent && (limit || one)) {
+          if (limit && limit.rows == null) {
```

Comment:
isn't this also an issue for pg and sqlite?

Reply:
This is already handled in the `super.limit` function, but was missing from the HANA specific expand `limit` logic.